packages/adminrouter/extra/src/.isort.cfg

```diff
@@ -0,0 +1,4 @@
+[settings]
+multi_line_output=3
+known_first_party=mocker,generic_test_code,runner,util,exceptions
```

Comment:
Did you happen to find why this line is necessary? If so, please document that here along with instructions on when to add  to these. One worry is that there are third party tools with the same names as our tools, such as https://pypi.python.org/pypi/mocker.